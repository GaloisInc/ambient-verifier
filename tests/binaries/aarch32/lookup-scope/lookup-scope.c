#include "ambient_assert.h"

char a();
char b();
char f();

int main() {
  ambient_assert(a() == 'a');
  ambient_assert(b() == 'b');
  ambient_assert(f() == a());
  return 0;
}
