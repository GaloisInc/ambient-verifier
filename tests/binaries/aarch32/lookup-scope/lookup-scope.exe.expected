successful: 0
failed: 0
overrideDir: tests/overrides/
sharedObjects: tests/binaries/aarch32/lookup-scope
